src/checkbox/index.js

```diff
@@ -20,7 +20,9 @@ const Checkbox = ({
     <label className={classNames}>
       <input
         type="checkbox"
+        name={label} // when name prop isn't passed, name sets to same value as label prop
```

Comment:
This doesn't allow customized name right ?

Reply:
The user can pass in the `name`  prop, which overwrites the default `name={label}`